appserver/payara-appserver-modules/rest-monitoring/rest-monitoring-war/pom.xml

```diff
@@ -71,6 +71,28 @@
             <version>7.0</version>
             <scope>provided</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>${mockito.version}</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>3.9.1</version>
+            <scope>test</scope>
+        </dependency>
+
+        <!-- JSONP Provider -->
+        <dependency>
+            <groupId>org.glassfish</groupId>
+            <artifactId>javax.json</artifactId>
+            <version>1.1</version>
```

Comment:
Why not use the same version and from jakarta as https://github.com/payara/Payara/blob/2e6edd27d6f1ac9837bf072eda4d794eae0bb04a/pom.xml#L186
seems to be used for example in https://github.com/payara/Payara/blob/2e6edd27d6f1ac9837bf072eda4d794eae0bb04a/api/payara-bom/pom.xml#L798-L802

Reply:
I though the `org.glassfish:jakarta.json` pointed to the API, and not an actual implementation., as only the API artifacts should be 'jakartified'.
Fixed in new commit.